reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md

```diff
@@ -110,44 +106,39 @@ Minimized, Maximized and Hidden.
 
 -Command
 Executes the specified commands (and any parameters) as though
-they were typed at the Windows PowerShell command prompt, and
+they were typed at the PowerShell command prompt, and
 then exits, unless the NoExit parameter is specified.
 
 The value of Command can be "-", a string. or a script block. If
 the value of Command is "-", the command text is read from standard
 input.
 
 Script blocks must be enclosed in braces ({}). You can specify a
-script block only when running PowerShell.exe in Windows PowerShell.
+script block only when running pwsh in PowerShell.
 The results of the script are returned to the parent shell as
 deserialized XML objects, not live objects.
 
 If the value of Command is a string, Command must be the last
 parameter in the command , because any characters typed after
 the command are interpreted as the command arguments.
 
-To write a string that runs a Windows PowerShell command, use the
+To write a string that runs a PowerShell command, use the
 format:
 "& {<command>}"
 where the quotation marks indicate a string and the invoke operator
 (&) causes the command to be executed.
 
 -Help, -?, /?
-Displays help for PowerShell.exe. If you are typing a PowerShell.exe
-command in Windows PowerShell, prepend the command parameters with a
+Displays help for pwsh. If you are typing a pwsh
+command in PowerShell, prepend the command parameters with a
 hyphen (-), not a forward slash (/). You can use either a hyphen or
 forward slash in Cmd.exe.
 
-# REMARKS:
-
-Troubleshooting note: In Windows PowerShell 2.0, starting some programs
-from the Windows PowerShell console fails with a LastExitCode of 0xc0000142.
-
 # EXAMPLES
 
-PowerShell -Version
+pwsh -Version
 
-PowerShell -Command {Get-EventLog -LogName security}
+pwsh -Command {Get-EventLog -LogName security}
```

Comment:
Perhaps we could use this opportunity to change this to a cross-platform example? `Get-Process -Name pwsh` maybe?